src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -203,7 +203,7 @@ def updateElements(self, *elementIds, **updatedParams):
         eleParams[self.eleKey] = updatedParams
         conflictIDs = self.db.updateBulkDocumentsWithConflictHandle(elementIds, eleParams, maxConflictLimit=20)
         if conflictIDs:
-            raise CouchConflictError("WQ update failed with conflict", data=updatedParams, result=conflictIDs)
+            raise CouchConflictError("WQ update failed with conflict", data=updatedParams, result=conflictIDs, status=400)
```

Comment:
Why did you hardcode the 400 exit status here?

Reply:
Because `status` is mandatory parameter in `CouchCobnflictError`, see https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Database/CMSCouch.py#L1147 and because in this call it was not provided. The pylint correctly complaint about it.

Reply:
I don't think it is mandatory, as there is a default value in the parent class:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Database/CMSCouch.py#L1097

The 400 does seem to be the best choice as well, because this is not an user query error, it's just a concurrency error inherited to distributed systems. I would suggest not to change it, unless you are 100% sure that it is required (which I would be surprised to see why it didn't pop up before).

Reply:
According to class definition:
```
class CouchConflictError(CouchError):
    def __init__(self, reason, data, result, status):
        CouchError.__init__(self, reason, data, result, status)
        self.type = "CouchConflictError"
```
the constructor of `CouchConflictError` does not have status as optional parameter, it is mandatory. Therefore, either we need change ctor of this class to make `status=None` or `status=0` (last is more correct from data-type point of view) or as I did  (and pylint spot it correctly) we must provide `status` value during ctor call.

Reply:
You are right!
Then please change it to the correct status code, which according to CMSCouch is meant to be `409`